core/locale/module.go

```diff
@@ -34,11 +34,13 @@ func (m *Module) DefaultConfig() config.Map {
 		"locale": config.Map{
 			"locale": "en-US",
 			"accounting": config.Map{
-				"decimal":    ",",
-				"thousand":   ".",
-				"formatZero": "%s -,-",
-				"format":     "%s %v",
-				"formatLong": "%v %v",
+				"default": config.Map{
+					"decimal":    ",",
```

Comment:
The default locale is `en-US`, so this should be `.`, right? As well as `,` for the thousands operator default

Reply:
yes makes sense - going to change